java/client/src/org/openqa/selenium/internal/Require.java

```diff
@@ -159,6 +159,17 @@ public static int positive(String argName, Integer number, String message) {
     return number;
   }
 
+  public static double positive(String argName, double number, String message) {
+    if (number <= 0) {
+      if (message == null) {
+        throw new IllegalArgumentException(argName + " must be greater than 0");
```

Comment:
Prefer adding a second `positive(String, double)` that delegates down to this three-param version. Using `null` in code is generally Not A Great Idea, and it looks ugly.